### ISSUE TITLE
Your player_node code crashes

### ISSUE THREAD

**Issue by miguelriemoliveira** [Tisila/rws2020_tavila#1]
Hi @Tisila ,

your player node is crashing. Here's the output:

```
File "/home/mike/catkin_ws/src/rws2020/rws2020_tavila/player_tavila/src/player_node.py", line 58
SyntaxError: Non-ASCII character '\xc2' in file /home/mike/catkin_ws/src/rws2020/rws2020_tavila/player_tavila/src/player_node.py on line 58, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for detail
```

Can you solve it and push?
Thanks

**Comment by Tisila** [comment]
Hi @miguelriemoliveira ,

I used unnecessary strange character. Already fixed it in commit #**c737ca7**.
Hope it builds and thanks for the issue. ;-)

**Comment by miguelriemoliveira** [comment]
Thanks @Tisila 

Your player is already there.

![image](https://user-images.githubusercontent.com/11350998/74990707-e29d8880-543b-11ea-9415-aa4555709898.png)


Closing issue!
